packages/amplify-codegen/src/commands/types.js

```diff
@@ -62,18 +62,27 @@ async function generateTypes(context, forceDownloadSchema, withoutInit = false,
           cwd: projectPath,
           absolute: true,
         });
-        const queries = queryFilePaths.map(queryFilePath => {
-          const fileContents = fs.readFileSync(queryFilePath, 'utf8');
-          if (
-            queryFilePath.endsWith('.jsx') ||
-            queryFilePath.endsWith('.js') ||
-            queryFilePath.endsWith('.tsx') ||
-            queryFilePath.endsWith('.ts')
-          ) {
-            return extractDocumentFromJavascript(fileContents, '');
-          }
-          return new Source(fileContents, queryFilePath);
-        });
+        const queries = queryFilePaths
+          .map(queryFilePath => {
+            const fileContents = fs.readFileSync(queryFilePath, 'utf8');
+            if (
+              queryFilePath.endsWith('.jsx') ||
+              queryFilePath.endsWith('.js') ||
+              queryFilePath.endsWith('.tsx') ||
+              queryFilePath.endsWith('.ts')
+            ) {
+              return [queryFilePath, extractDocumentFromJavascript(fileContents, '')];
+            }
+            return [queryFilePath, new Source(fileContents, queryFilePath)];
+          })
+          .filter(([queryFilePath, source]) => {
+            if (!source) {
+              context.print.warning(`Unable to extract GraphQL queries from ${queryFilePath}. Skipping source.`);
```

Comment:
We could also add something like:

```
This file matched the includes target in .grapqhlconfig.yml. Modify the includes or excludes target if this file should not be included.
```

Reply:
Yeah, I think a CTA here would be helpful, in addition to the error message.